src/canvas/SelectableCanvas.ts

```diff
@@ -1104,6 +1104,11 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
       return false;
     }
     this._activeObject = object;
+    if (object instanceof ActiveSelection) {
+      this._activeSelection = object;
+      this._activeSelection.set('canvas', this);
+      this._activeSelection.setCoords();
+    }
```

Comment:
this is good for me, give me the extra happyness to save some bytes by doing:

```suggestion
    if (object instanceof ActiveSelection) {
      this._activeSelection = object;
      object.set('canvas', this);
      object.setCoords();
    }
```